src/stream-analytics/azext_stream_analytics/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": true,
-    "azext.minCliCoreVersion": "2.0.67"
+    "azext.minCliCoreVersion": "2.3.1"
```

Comment:
Why 2.3.1

Reply:
+1

Reply:
`is_experimental` in introduced from 2.3.0. And as there are some breaking bugs in 2.3.0, so I use 2.3.1 here.